lib/src/disposable/disposable.dart

```diff
@@ -427,6 +428,28 @@ class Disposable implements _Disposable, DisposableManagerV3 {
     return null;
   }
 
+  void subscribeToDocumentEvent(String event, EventListener callback,
+      {bool useCapture: null, EventTarget documentObject: null}) {
+    if (documentObject == null) {
+      documentObject = document;
+    }
+    _subscribeToEvent(documentObject, event, callback, useCapture);
+  }
+
+  void subscribeToDomElementEvent(
+      Element element, String event, EventListener callback,
+      {bool useCapture: null}) {
+    _subscribeToEvent(element, event, callback, useCapture);
+  }
+
+  void subscribeToWindowEvent(String event, EventListener callback,
+      {bool useCapture: null, EventTarget windowObject: null}) {
```

Comment:
I don't know if I'd support a `null` default value for a boolean. It seems like the strategy is to defer the default value to whatever `eventTarget.addEventListener` decides. If a default value is undecided, I would suggest a doc comment explaining how a default is arrived and why you may want to choose something other than the default.